crates/typst/src/math/mod.rs

```diff
@@ -128,6 +128,9 @@ use crate::text::TextElem;
 /// because just one `&` precedes it. Each alignment point simply alternates
 /// between right-aligned/left-aligned.
 ///
+/// By default, there is a gap of `1.5em` added between columns. You can
+/// modify this with the [`column-gap`]($math.equation.column-gap) parameter.
+///
 /// ```example
 /// $ (3x + y) / 7 &= 9 && "given" \
 ///   3x + y &= 63 & "multiply by 7" \
```

Comment:
To better showcase the gap added, we could change this example to
```typ
$ (3x + y) / 7 &= 9        &                &"given" \
        3x + y &= 63       & "multiply by 7"& \
            3x &= 63 - y   &                &"subtract y" \
             x &= 21 - y/3 &   "divide by 3"& $
```
![test](https://github.com/user-attachments/assets/675b50c4-31b7-4d55-b9bb-3e164b877338)